src/medium-editor-md.js

```diff
@@ -26,10 +26,12 @@ module.exports = function (options, callback) {
     options.events = options.events || ["input", "change"];
     callback = callback || options.callback || function () {};
 
-    // Wait until the Medium Editor is inited
-    setTimeout(function() {
+    // Called by medium-editor during init
+    this.init = function(mediumEditorInstance) {
```

Comment:
We can actually get rid of `self` and just use `this` since we're not inside a `setTimeout` anymore, I just didn't want to make too many changes and cause the intent of the pull request to get lost.  I can make an update then.

Also, since the instance of medium-editor will contain the elements we want, it seems like we should just always use the elements in the `.elements` array right?  I ask, because if we do that, and don't take in the element as an option, then we can get rid of the ternary operator altogether.

I'll make the other changes as well.